handlers/product-move-api/ProductMovementAPI.yaml

```
@@ -118,11 +119,11 @@ components:
         - currency
       properties:
         amount:
-          type: number
+          type: integer
           description:
-            Absolute amount that will be billed.
+            Absolute amount that will be billed in pence.
```

I don't know what the generic term for pence, cents, etc is.
```suggestion
            Absolute amount that will be billed in pence or cents etc.
```